src/pages/studyView/table/StructuralVariantMultiSelectionTable.tsx

```diff
@@ -0,0 +1,782 @@
+                            }
+                        </div>
+                    );
+                },
+                render: (data: StructVarMultiSelectionTableRow) => (
+                    <span
+                        data-test={'numberOfAlterations'}
+                        style={{
+                            flexDirection: 'row-reverse',
+                            display: 'flex',
+                            marginRight: cellMargin,
+                        }}
+                    >
+                        {data.totalCount.toLocaleString()}
+                    </span>
+                ),
+                sortBy: (data: StructVarMultiSelectionTableRow) =>
+                    data.totalCount,
+                defaultSortDirection: 'desc' as 'desc',
+                filter: (
+                    data: StructVarMultiSelectionTableRow,
+                    filterString: string
+                ) => {
+                    return _.toString(data.totalCount).includes(filterString);
+                },
+                width: columnWidth,
+            },
+        };
+        return defaults[columnKey];
+    };
+
+    getDefaultCellMargin = (
+        columnKey: StructVarMultiSelectionTableColumnKey,
+        columnWidth: number
+    ) => {
+        const defaults: {
+            [key in StructVarMultiSelectionTableColumnKey]: number;
+        } = {
+            [StructVarMultiSelectionTableColumnKey.GENE1]: 0,
+            [StructVarMultiSelectionTableColumnKey.GENE2]: 0,
+            [StructVarMultiSelectionTableColumnKey.STRUCTVAR_SELECT]: 0,
+            [StructVarMultiSelectionTableColumnKey.NUMBER_STRUCTURAL_VARIANTS]: correctMargin(
+                getFixedHeaderNumberCellMargin(
+                    columnWidth,
+                    this.totalCountLocaleString
+                )
+            ),
+            [StructVarMultiSelectionTableColumnKey.NUMBER]: correctMargin(
+                (columnWidth -
+                    10 -
+                    (getFixedHeaderTableMaxLengthStringPixel(
+                        this.alteredCasesLocaleString
+                    ) +
+                        30)) /
+                    2
+            ),
+            [StructVarMultiSelectionTableColumnKey.FREQ]: correctMargin(
+                getFixedHeaderNumberCellMargin(
+                    columnWidth,
+                    getFrequencyStr(
+                        _.max(
+                            this.tableData.map(
+                                item =>
+                                    (item.numberOfAlteredCases! /
+                                        item.numberOfProfiledCases!) *
+                                    100
+                            )
+                        )!
+                    )
+                )
+            ),
+        };
+        return defaults[columnKey];
+    };
+
+    @computed
+    get maxNumberTotalCount() {
+        return _.max(this.tableData.map(item => item.totalCount));
```

Comment:
this is a tiny quibble but ```_.maxBy(this.tableData, item => item.totalCount).totalCount```
is more efficient i think

Reply:
Updated!